Portal/vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/client/view/ssh/EditSSHLayout.java

```diff
@@ -87,7 +93,21 @@ private void configure() {
         portField = FieldUtil.getTextItem(450, null);
         directoryField = FieldUtil.getTextItem(450, null);
         statusField = FieldUtil.getTextItem(450, null);
-        
+        numberSynchronizationFailedField = FieldUtil.getTextItem(450, null);
```

Comment:
Disable this field and earliestNextSynchronization date. Add a redmine ticket to reset these 2 fields.